src/app/components/Table/index.tsx

```diff
@@ -105,7 +105,7 @@ export const Table: FC<TableProps> = ({
             <TableRow>
               {columns.map((column, index) => (
                 <TableCell
-                  key={column.content}
+                  key={`field-${index}`}
```

Comment:
From discussion: this might be risky

I would rather follow the pattern from table body where rows and cells have `key` field

Reply:
> I would rather follow the pattern from table body where rows and cells have key field

Updated to do this.